civihr_employee_portal/features/node_export_files/mydetails_personal_details.export

```diff
@@ -220,16 +220,19 @@ array(
             'value' => '',
             'extra' => array(
               'items' => "1|Female\n2|Male\n3|Other",
-              'aslist' => 1,
+              'multiple' => 0,
+              'title_display' => 'before',
+              'description_above' => 0,
               'private' => 0,
-              'multiple' => NULL,
+              'wrapper_classes' => '',
+              'css_classes' => '',
+              'aslist' => 1,
+              'civicrm_live_options' => '1',
```

Comment:
I would expect this to be the only change in this file, but I can actually see a lot more updates for other options. @powdevel can you confirm that this is not changing anything we don't want to?

Reply:
@davialexandre This PR also removes NI/SSN fields. See this link: https://compucorp.atlassian.net/browse/PCHR-3967?focusedCommentId=73007&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-73007

Reply:
@powdevel I know that, but this does not answer my question. I'm talking about the changes in the 0a17d3eaa9bad3b9e1e75912f684ef12f89b4606 which are only about "setting the live option when exporting the webform". If that was the only thing done in that commit, then why we have all of these other changes it in?

Reply:
@davialexandre that is what the export returned. I think the changes you are looking are related to field that was changed, but as webform has many sections, the form, the notifications, the civicrm, etc one simple change will update more than one section.